src/extension.ts

```diff
@@ -1282,6 +1283,16 @@ async function createClusterKubernetes(request? : WizardUIRequest) {
     }
 }
 
+async function useContextKubernetes(explorerNode: explorer.ResourceNode) {
+    const targetContext = explorerNode.metadata.context;
+    const shellResult = await kubectl.invokeAsync(`config use-context ${targetContext}`);
+    if (!shellResult.code) {
+        vscode.commands.executeCommand("extension.vsKubernetesRefreshExplorer");
+    } else {
+        vscode.window.showErrorMessage(`set current cluster failed: ${shellResult.stderr}`);
```

Comment:
Error messages should be sentence cased.

Reply:
nice. fixed.